### PR TITLE
Initial commit of ENSO analysis

### PR DESCRIPTION
**Pull request by jkrasting**

### CODE REVIEW

**Comment by raphaeldussin** [comment]
looks like we're missing xwavelet in the environment file. do you have it published on pip?

**Comment by jkrasting** [comment]
> looks like we're missing xwavelet in the environment file. do you have it published on pip?

Not yet - still kicking the tires on it.  I've added it to the CI environment via git.

File: om4labs/diags/enso/enso.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+    Returns
+    -------
+    (xarray.DataArray, xarray.DataArray)
+    """
+
+    array = xr.open_mfdataset(dictArgs["infile"], use_cftime=True)[dictArgs["varname"]]
+    areacello = xr.open_dataset(dictArgs["static"])["areacello"].fillna(0.0)
+
+    array.load()
```
Comment (raphaeldussin) [inline om4labs/diags/enso/enso.py:96]: do we need to load into memory already?

**Review by raphaeldussin — APPROVED** [review]

**Comment by jkrasting** [comment]
Good catch.  Probably not.

**Review by raphaeldussin — APPROVED** [review]

**Comment by raphaeldussin** [comment]
we should provide a notebook to create the observation file `wavelet.NOAA-ERSST-v5.1957-2002.nc`

**Comment by raphaeldussin** [comment]
beautiful!

**Pull request merged**